extensions/roles/ledger_roles_config.go

```diff
@@ -25,3 +28,13 @@ func IsValidator() bool {
 func RolesAsString() []string {
 	return []string{}
 }
+
+// HasEndorserRole returns true if given set of roles has endorser role
```

Comment:
Are these functions used anywhere?

Reply:
forgot to clean this up.
Used in fabric-peer-ext, will remove these in fabric-mod and will add them in fabric-peer-ext

Reply:
deleted in fabric-mod